keyboards/neito/keymaps/olli.works/keymap.c

```diff
@@ -0,0 +1,64 @@
+/* Copyright 2020 Oliver Granlund
```

Comment:
The `.` in this keymap name may need to be changed to an `_`, I'm not sure how well the build system/CLI will take it.